ecowitt2mqtt/helpers/device.py

```diff
@@ -17,6 +17,7 @@
     "GW2000A": ("Ecowitt", "GW2000A"),
     "GW2000B": ("Ecowitt", "GW2000B"),
     "PT-HP2550": ("Fine Offset", "HP2550"),
+    "HP2550_Pro": ("Misol", "HP2250_Pro"),
```

Comment:
Small not: please keep the keys in alphabetical order.

Reply:
Done